public/assets/styles/custom.css

```diff
@@ -282,6 +282,66 @@ section.layers .limited {
 }
 
 #mint-nft {
-  margin: 10px 20px;
+  margin: 10px 0;
   min-width: 120px;
+}
+
+.mint-info {
+  margin: 10px;
+}
+
+.mintH1{
+  letter-spacing: 3px;
+    -webkit-text-stroke: white;
+    -webkit-text-stroke-color: #fffa;
+    -webkit-text-stroke-width: 2px;
+}
+.mint-info-container{
+  display: flex;
+    flex-direction: column;
+    align-items: stretch;
+}
+.mint-info>div{
+  opacity:0;
+  transform:translateY(20px);
+  animation: details-fade-in 1s ease-out forwards;
+  background:#0001;
+  text-shadow:0 0 2px white;
+}
+.mint-info>div:nth-child(1){
```

Comment:
@levan-k we can speed this up and set to increments of 0.25 instead of 0.5 too. I wasn't sure which I liked better. Tested on mobile and PC